test/scr_parse_test.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+RSpec.describe "slp.conf parsing" do
+  COMMENTED_LINES = File.join(FIXTURES_PATH, "commented-slp.conf")
+  UNCOMMENTED_LINES = File.join(FIXTURES_PATH, "uncommented-slp.conf")
+
+  COMMENTED_REGEXP = Regexp.new '^[#;][ \t]*([^ \t\=]+)[ \t\=]*[ ]*(.+)[ \t]*$'
+  UNCOMMENTED_REGEXP = Regexp.new '[ \t]*([^ \t\=]+)[ \t\=]+[ ]*(.+)[ \t]*$'
```

Comment:
I think this should work: move `test/fixtures/{un,}commented-slp.conf` to `test/fixtures/{un,}commented}/etc/slp.conf`, that is, treat them like the `valid` and `broken` cases, then `change_scr_root(fixtures); SCR.Read(path(".etc.slp.all"))` and check that.